src/d3a/d3a_core/sim_results/area_statistics.py

```diff
@@ -327,19 +330,53 @@ def generate_inter_area_trade_details(area, past_market_types):
     return trade_details
 
 
+def generate_area_cumulative_trade_redis(child, accumulated_trades):
+    results = {"areaName": child.name}
+    area_data = accumulated_trades[child.name]
+    # Producer entries
+    results["bars"] = []
+    if abs(area_data["produced"]) > FLOATING_POINT_TOLERANCE:
+        results["bars"].append(
+            {"energy": area_data["produced"], "targetArea": child.name,
+             "energyLabel":
+                 f"{child.name} Produced {str(round(abs(area_data['produced']), 3))} kWh",
+             "priceLabel":
+                 f"{child.name} Earned {str(round(abs(area_data['earned']), 3))} cents"}
+        )
+    if child.name in area_data["consumedFrom"]:
+        energy = area_data["consumedFrom"][child.name]
+        money = area_data["spentTo"][child.name]
+        results["bars"].append({
+            "energy": energy,
+            "targetArea": child.name,
+            "energyLabel": f"{child.name} Consumed {str(round(energy, 3))} kWh from {child.name}",
```

Comment:
Just very minor but is bugging me since this plot was added to the fronend: Why capital "Consumed" (and also Produced, Earned, Spent) ? I read the string as a sentence and consequently the first letter of this verb has to be lowercase, right?

Reply:
Because Javascript uses mostly CamelCase format on its code, and since this code is mostly for consumption for the frontend, it makes sense to have these values in this format, instead of snake case that Python uses. At least this was the reason why we originally used camel case for these.

Reply:
Sorry, i want clear. I did not mean the key string. I mean the value string.

Reply:
Ahhhhh now I see it ! You are correct, I am fixing this. Thanks!